nixos/modules/services/hardware/vdr.nix

```diff
@@ -45,6 +45,7 @@ in {
     systemd.services.vdr = {
       description = "VDR";
       wantedBy = [ "multi-user.target" ];
+      after = [ "network.target" ];
```

Comment:
Do you mean `network.target` or rather `network-online.target`?
See https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/

Reply:
network.target, we should guaranty, that all network cards are initialized and during shutdown, the service is stopped before network cards going down.